themes/theme-gmd/pages/Cart/components/Item/components/Product/components/Layout/components/Title/style.js

```diff
@@ -9,25 +9,29 @@ const title = css({
 }).toString();
 
 const menuContainer = css({
+  marginTop: `-${variables.gap.bigger}px`,
   marginRight: `-${variables.gap.big}px`,
-  marginLeft: variables.gap.big,
-  pointerEvents: 'none',
 }).toString();
 
-const menuToggleSize = variables.gap.big * 2;
+const menuToggleSize = variables.gap.big * 2.5;
 const menuToggleFontSize = variables.gap.big * 1.5;
 
-const menuToggle = css({
+const menuToggleContainer = css({
+  margin: variables.gap.small,
+}).toString();
```

Comment:
Please remove the `.toString()` function call here. This is not needed in this case and can cause weird warnings and issues. This technique is ancient and not wanted anymore. And yes, `classNames` can still deal with it ;)

Reply:
See comment at the bottom. The `classnames` module expects strings as input